CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an issue where possessing a new pawn and immediately setting the owner of the old pawn to the controller could cause server RPCs from that pawn to be dropped.
 - Added support for the `bHidden` relevancy flag. Clients will not checkout Actors that have `bHidden` set to true (unless they are always relevant or the root component has collisions enabled).
 - Fixed an issue with deployments failing due to the incorrect number of workers when the launch config was specified, rather than automatically generated.
+- Fixed the `too many dynamic subobjects` error. Added the `RemovedDynamicSubobjectObjectRefs` map in `USpatialPackageMapClient` that keeps the dynamic subobjects removed from Startup Actor's clients interest to avoid dublication of the dynamic subojects when the Startup Actor re-enters the Client's interest.
```

Comment:
Isn't this specifically when moving in and out of interest / boundary. I'd suggest we be more specific because there are genuine times when a dev uses too many dynamic subobjects of the same type for a schema setting